cron/poll_read_issue_comments.py

```diff
@@ -145,6 +178,48 @@ def handle_vote_command(api, command, issue_id, comment_id, votes):
             gh.issues.close_issue(api, settings.URN, issue_id)
         elif sub_command == "reopen":
             gh.issues.open_issue(api, settings.URN, issue_id)
+        elif sub_command == "fast":
+            comment_updated_at = cmdmeta.comment.updated_at
+            comment_created_at = cmdmeta.comment.created_at
+            comment_poster = cmdmeta.comment.user
+            issue_poster = cmdmeta.issue.user
+            issue_created_at = cmdmeta.issue.created_at
+
+            # This must be a PR
+            if not cmdmeta.issue.is_pr:
+                return
+
+            # The post should not have been updated
+            if updated_at != created_at:
```

Comment:
maybe leave some kind of response indicating why the command didn't run? Makes debugging easier